ksql-rest-app/src/main/java/io/confluent/ksql/rest/util/StructSerializationModule.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.confluent.ksql.rest.util;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import java.io.IOException;
+import java.util.Collections;
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.json.JsonConverter;
+
+public class StructSerializationModule extends SimpleModule {
+
+  private static final JsonConverter jsonConverter = new JsonConverter();
+
+  public StructSerializationModule(final ObjectMapper objectMapper) {
+    super();
+    jsonConverter.configure(Collections.singletonMap("schemas.enable", false), false);
+    addSerializer(Struct.class, new StructSerializationModule.Serializer(objectMapper));
+  }
+
+  static class Serializer extends JsonSerializer<Struct> {
+
+    private final ObjectMapper objectMapper;
+
+    public Serializer(final ObjectMapper objectMapper) {
+      this.objectMapper = objectMapper;
+    }
+
+    @Override
+    public void serialize(
+        final Struct struct,
+        final JsonGenerator jsonGenerator,
+        final SerializerProvider serializerProvider
+    ) throws IOException {
+      struct.validate();
+      jsonGenerator.writeRaw(
+          new String(jsonConverter.fromConnectData("", struct.schema(), struct)));
```

Comment:
don't use default encoding. Use `new String(bytes, StandardCharset.UTF_8)`

Reply:
I still personally think this is much more risky that just using the code I supplied. I really don't see the benefit of using another Json serializer over the 5 lines of code I showed. But... what ever! ;)

Reply:
Using or not using Connect converter needs more discussion. Let's have this as is for this PR and decide about if we should use Connect Convertor later.